docs/framework/windows-workflow-foundation/net-framework-4-5-built-in-activity-library.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: ".NET Framework 4.5 Built-In Activity Library"
+description: This article describes the new activity library with expanded functionality that is part of .NET Framework 4.6.1.
```

Comment:
```suggestion
description: This article describes the new activity library with expanded functionality that is part of .NET Framework 4.5.
```

Also, the include on line 10 should be removed and replaced with .NET Framework 4.5.